### PR TITLE
mosman94109-Patch 1

### PR DESCRIPTION
**Pull request by mosman94109**

### CODE REVIEW

File: docs/support/index.rst [hunk 1]
```diff
@@ -134,6 +135,18 @@ ____________________________________
...
+The transport argument
+______________________
+
+Certain drivers support providing an alternate transport in the :code:`optional_args`, overriding the default protocol to connect with. Allowed transports are therefore device/library dependant:
+
+=============== ====================  ====================  ===================
+_               EOS                   NXOS                  IOS
+=============== ====================  ====================  ===================
+**Default**     ``https``             ``https``             ``ssh``
+**Supported**   ``http``, ``https``   ``http``, ``https``   ``telnet``, ``ssh``
```
Comment (ktbyers) [inline docs/support/index.rst]: telnet is not currently supported on IOS (there is a PR open on it).

**Comment by dbarrosop** [comment]
@mosman94109 looks like you created 9 patches. Would you mind reviewing them and closing the ones that are duplicated?

**Comment by mosman94109** [comment]
@dbarrosop, not sure what you mean. Each patch is for a different file.

**Comment by dbarrosop** [comment]
All your PRs contain a common commit that is making everything confusing. A single PR with all your changes and without introducing someone else's commit would be preferable.

Could you, please, close all the PRs and send a new one with all the changes? Also, make sure you don't introduce someone else's commit.

**Pull request closed without merging**